alchemy/src/scope.ts

```diff
@@ -466,6 +477,8 @@ export class Scope {
     });
 
     if (!this.parent && process.env.ALCHEMY_TEST_KILL_ON_FINALIZE) {
+      this.logger.log("[debug] Killing on finalize");
+      await Promise.allSettled(this.cleanups.map((cleanup) => cleanup()));
```

Comment:
Should we provide a function that does this since it's done in multiple places?

`cleanup()`

Reply:
Sure